metrics/Inactive_Contributers.md

```diff
@@ -0,0 +1,33 @@
+# Inactive_Contributers
```

Comment:
"Contributors" is misspelled. Also, the name of the metric has no underscore, though the file does.